go/src/koding/go-webserver/envdata.go

```diff
@@ -17,10 +17,10 @@ import (
 const myPermissionsAndRolesPath = "/-/my/permissionsAndRoles"
 
 var defClient = httputil.NewClient(&httputil.ClientConfig{
-	DialTimeout:           1 * time.Second,
-	RoundTripTimeout:      2 * time.Second,
-	TLSHandshakeTimeout:   2 * time.Second,
-	ResponseHeaderTimeout: 2 * time.Second,
+	DialTimeout:           30 * time.Second,
```

Comment:
i dont know why you increased these but 30 secs a bit high since we terminate the conn after 20 secs on ELB.
10 secs could be a better fit than 30. 

wdyt @rjeczalik ?